README.md

```diff
@@ -1,22 +1,99 @@
-# carpools
-
-## Running
+# Nomad
 
+## Getting Started
 1. Check out the code
 
-   ```
-   git clone git@github.com:iandees/carpools.git
-   ```
+    ```
+    git clone git@github.com:RagtagOpen/nomad.git
+    ```
+
+1. Go into the checked out repository
+  
+    ```bash
+    cd nomad
+    ```
+
+1. Local Variables
+    
+    Create a file for your local environment variables. This file should be called `.env` and live at the root of the nomad project directory.
+    
+    ```bash
+    touch .env
+    echo FLASK_APP=wsgi.py >> .env
+    echo FLASK_DEBUG=1 >> .env
+    ```
+    
+1. (Optional) Get a Google Maps Api Key
+    
+    In order to run the search frontend you will need an API key for Google maps. You can get one [here](https://developers.google.com/maps/documentation/javascript/get-api-key).
+    Then set the key as a variable in your `.env` file.
+    
+    ```bash
+    echo GOOGLE_MAPS_API_KEY="YOUR_KEY_HERE" >> .env
+    ```
+    
+1. (Optional) Configure Google Sign-In
+    In order to test Google authentication you'll need to create a [Console Project](https://developers.google.com/identity/sign-in/web/devconsole-project).
+    After going through the project creation step you'll have an auto-generated project ID. This needs to go in your `.env` file.
```

Comment:
I don't think we need the app ID separate from the Oauth client ID. Can you delete this section?